SuperBuild/External_vtkIECTransformLogic.cmake

```diff
@@ -71,7 +71,7 @@ if(NOT DEFINED ${proj}_DIR AND NOT ${CMAKE_PROJECT_NAME}_USE_SYSTEM_${proj})
       -DCMAKE_LIBRARY_OUTPUT_DIRECTORY:PATH=${CMAKE_BINARY_DIR}/${Slicer_THIRDPARTY_LIB_DIR}
       -DCMAKE_ARCHIVE_OUTPUT_DIRECTORY:PATH=${CMAKE_ARCHIVE_OUTPUT_DIRECTORY}
       # Install directories
-      # NA
+      -DvtkIECTransformLogic_INSTALL_LIB_DIR:PATH=${Slicer_QTLOADABLEMODULES_LIB_DIR}
```

Comment:
Third-party shared libraries should not be installed in a module folder. It may work on some operating systems, but probably not on all.

As I wrote in the referenced issue, please follow a working example, such as the SlicerOpenIGTLink extension:

https://github.com/openigtlink/SlicerOpenIGTLink/blob/6fbdadf16d6ccee8e840d9d408422bec4c95e867/SuperBuild/External_OpenIGTLink.cmake#L73